### PR TITLE
hw6

### PR DESCRIPTION
**Pull request by Artemijacd**
SOLID Principles

### CODE REVIEW

File: docs/6-solid-principles/hometask.md [hunk 1]
```diff
@@ -10,13 +10,16 @@ sidebar_position: 6
...
-   | Liskov Substitution Principle    |          |
-   | Interface Seggregation Principle |          |
-   | Dependency Inversion Principle   |          |
+https://github.com/microsoft/vscode/tree/main/src
+
+
+   | Principle                        |                                    Examples                                        |
+   | -------------------------------- |                                    --------                                        |
+   | Single Responsibility Principle  |   src/vs/base/browser/browser.ts; line: 9; class: WindowManager                    |
+   | Open / Closed Principle          |   src/vs/base/browser/ui/button/button.ts; line: 45; class: IButton                |
```
Comment (metam0rphus) [inline docs/6-solid-principles/hometask.md:19]: it's interface

File: docs/6-solid-principles/hometask.md [hunk 2]
```diff
@@ -25,3 +28,12 @@ sidebar_position: 6
    You may provide short descriptions about how to refactor/improve such violations.
 
    Optionally, you can add small examples with results of such refactoring using pseudocode or real code.
+
+
+   | Principle                        |                                    Examples                                                      |
+   | -------------------------------- |                                    --------                                                      |
+   | Single Responsibility Principle  |   src/vs/code/electron-sandbox/processExplorer/processExplorerMain.ts; class: FastDomNode        |
+   | Open / Closed Principle          |   src/vs/code/electron-sandbox/processExplorer/processExplorerMain.ts: line: 317; method: getId  |
+   | Liskov Substitution Principle    |                                                                                                  |
```
Comment (metam0rphus) [inline docs/6-solid-principles/hometask.md:37]: try to find class that doesn't implement all the interface's methods

File: docs/6-solid-principles/hometask.md [hunk 3]
```diff
@@ -25,3 +28,12 @@ sidebar_position: 6
...
+   | Interface Seggregation Principle |                                                                                                  |
```
Comment (metam0rphus) [inline docs/6-solid-principles/hometask.md:38]: try to find class that doesn't implement all the interface's methods

File: docs/6-solid-principles/hometask.md [hunk 4]
```diff
@@ -25,3 +28,12 @@ sidebar_position: 6
    You may provide short descriptions about how to refactor/improve such violations.
 
    Optionally, you can add small examples with results of such refactoring using pseudocode or real code.
+
+
+   | Principle                        |                                    Examples                                                      |
+   | -------------------------------- |                                    --------                                                      |
+   | Single Responsibility Principle  |   src/vs/code/electron-sandbox/processExplorer/processExplorerMain.ts; class: FastDomNode        |
```
Comment (metam0rphus) [inline docs/6-solid-principles/hometask.md:35]: there is no such class

File: docs/6-solid-principles/hometask.md (hunk 1 above)
Comment (Artemijacd) [reply docs/6-solid-principles/hometask.md]: fixed

**Pull request merged**